lib/exception_handling/exception_info.rb

```diff
@@ -249,6 +249,7 @@ def dump_hash( h, indent_level = 0 )
     def enhanced_data_to_honeybadger_context
       data = enhanced_data.dup
       data[:server] = ExceptionHandling.server_name
+      data[:context] = @exception_context if @exception_context.present?
```

Comment:
Probably, it is a little weird.  In the web side we call this param exception_context.